manifests/0000_26_cloud-controller-manager-operator_19_networkpolicy.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: ccmo-network-policy
+  namespace: openshift-cloud-controller-manager-operator
+spec:
+  podSelector: {}
+  policyTypes:
+    - Ingress
+    - Egress
+  ingress:
+    - from:
+      - podSelector:
+          matchLabels:
+            apiserver: 'true'
+            prometheus: k8s
```

Comment:
A match labels will need to match all labels, so I expect currently this will match no pods at all, you may need multiple entries in the `-from` list